src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -63,14 +69,21 @@ def load_arguments(self, _):
         c.argument('top', arg_type=top_arg_type)
         c.argument('all_', options_list=['--all'], action='store_true', help="List all items.")
         c.argument('fields', arg_type=fields_arg_type)
+        c.argument('sku', help='The sku of App Configuration', arg_type=get_enum_type(['free', 'standard']))
 
     with self.argument_context('appconfig create') as c:
         c.argument('location', options_list=['--location', '-l'], arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
-        c.ignore('sku')
+        c.argument('assign_identity', arg_type=identities_arg_type)
 
     with self.argument_context('appconfig update') as c:
         c.argument('tags', arg_type=tags_type)
 
+    with self.argument_context('appconfig identity assign') as c:
+        c.argument('identities', arg_type=identities_arg_type)
+
+    with self.argument_context('appconfig identity remove') as c:
+        c.argument('identities', arg_type=identities_arg_type, help="Accept system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity, '[all]' for all identities or a resource id to refer user assigned identity. Default: '[system]'")
```

Comment:
please declare your default value using `default="[system]"` for you argument or argument type not in help message?

Reply:
Same with "appconfig identity assign" command, the actual default value to fit with method remove_managed_identity should be ["[system]"] instead of "[system]" as --identities is a list instead of a string. From customer's perspective, if nothing specified it just means system assigned identity. So the best practice where maybe is just saying ```"Use system assigned identity if not specified."``` in the help message.